internal/pkg/template/workload.go

```diff
@@ -288,17 +282,42 @@ func ImportedVariable(name string) Variable {
 func (cfg *NetworkLoadBalancer) Aliases() []string {
 	var uniqueAliases []string
 	seen := make(map[string]bool)
-	for _, listener := range cfg.Listener {
-		for _, entry := range listener.Aliases {
+	for _, rule := range cfg.Listener {
```

Comment:
NLB's `cfg.Listener` contains a list of listeners, not a list of rules. NLB's listener doesn't have rules. Can we change the variable name back to `listener`?